app/lib/types.rb

```diff
@@ -74,4 +74,19 @@ module Types
     applicant_name
   ].freeze
   SortBy = String.default('submitted_at'.freeze).enum(*SORTABLE_COLUMNS)
+
+  FILE_TYPES = {
```

Comment:
Might be simplified by doing `File.extname(filename)`?
Just an idea. As it seems the important thing is the extension of the file.

Reply:
Thanks, i've added that